src/main/resources/assets/blast/lang/fr_fr.json

```diff
@@ -1,11 +1,70 @@
+  "item.blast.pearl_trigger_bomb": "Bombe de perle à détente",
+  "item.blast.amethyst_bomb": "Bombe en améthyste",
+  "item.blast.amethyst_trigger_bomb": "Bombe en améthyste à détente",
+  "item.blast.frost_bomb": "Bombe glacée",
+  "item.blast.frost_trigger_bomb": "Bombe glacée à détente",
+  "item.blast.slime_bomb": "Bombe gluante",
+  "item.blast.slime_trigger_bomb": "Bombe gluante à détente",
+  "item.blast.pipe_bomb": "Bombe artisanale",
+  "entity.blast.bombard": "Bombarde",
+  "entity.blast.slime_bombard": "Bombarde gluante",
+  "entity.blast.amethyst_bombard": "Bombarde en améthyste",
+  "entity.blast.frost_bombard": "Bombarde glacée",
+  "entity.blast.bomb": "Bombe",
+  "entity.blast.trigger_bomb": "Bombe à détente",
+  "entity.blast.golden_bomb": "Bombe en or",
+  "entity.blast.golden_trigger_bomb": "Bombe à détente en or",
+  "entity.blast.diamond_bomb": "Bombe en diamant",
+  "entity.blast.diamond_trigger_bomb": "Bombe à détente en diamant",
+  "entity.blast.naval_mine": "Mine navale",
+  "entity.blast.confetti_bomb": "Bombe à confettis",
+  "entity.blast.confetti_trigger_bomb": "Bombe à confettis à détente",
+  "entity.blast.dirt_bomb": "Bombe de terre",
+  "entity.blast.dirt_trigger_bomb": "Bombe de terre à détente",
+  "entity.blast.pearl_bomb": "Bombe de perle",
+  "entity.blast.pearl_trigger_bomb": "Bombe de perle à détente",
+  "entity.blast.amethyst_bomb": "Bombe en améthyste",
+  "entity.blast.amethyst_trigger_bomb": "Bombe en améthyste à détente",
+  "entity.blast.frost_bomb": "Bombe glacée",
+  "entity.blast.frost_trigger_bomb": "Bombe glacée à détente",
+  "entity.blast.slime_bomb": "Bombe gluante",
+  "entity.blast.slime_trigger_bomb": "Bombe gluante à détente",
+  "entity.blast.gunpowder_block": "Bloc de poudre à canon",
+  "entity.blast.stripminer": "Lamineur",
+  "entity.blast.cold_digger": "Fouilleur gelé",
+  "entity.blast.bonesburrier": "Enterreur d’os",
+  "entity.blast.amethyst_shard": "Éclat d’améthyste",
+  "entity.blast.icicle": "Stalactite",
+  "entity.blast.pipe_bomb": "Bombe artisanale",
+  "entity.blast.shrapnel_block": "Bloc d’éclats",
   "block.blast.gunpowder_block": "Bloc de poudre à canon",
-  "block.blast.stripminer": "Lamineur"
+  "block.blast.stripminer": "Lamineur",
+  "block.blast.cold_digger": "Fouilleur gelé",
+  "block.blast.remote_detonator": "Détonateur à distance",
+  "block.blast.bonesburrier": "Enterreur d’os",
+  "block.blast.dry_ice": "Glace sèche",
+  "block.blast.folly_red_paint": "Peinture rouge folie",
+  "block.blast.fresh_folly_red_paint": "Peinture rouge folie fraîche",
+  "block.blast.dried_folly_red_paint": "Peinture rouge folie séchée",
+  "death.attack.amethyst_shard": "%1$s a été poignardé par un éclat d’améthyste provenant de la bombe en améthyste de %2$s",
+  "death.attack.icicle": "%1$s a été glacé par %2$s",
+  "subtitles.blast.entity.pipe_bomb.tick": "La bombe artisanale fait tic-tac",
+  "subtitles.blast.entity.pipe_bomb.explode": "La bombe artisanale explose"
```

Comment:
How does it work for other explosion subtitles ? Is it "La bombe" or "Une bombe" ?